### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by czpython**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,95 +1,26 @@
...
-django-cms-search
-=================
+#############
+Aldryn Search
+#############
 
-This package provides multilingual search indexes for easy Haystack integration
-with `django CMS <http://www.django-cms.org>`_.
+Aldryn Search is an application that acts like a bridge between django-haystack and django-cms.
+It's also a great tool to hookup search capabilities to any django app, specially apps that make use of
```
Comment (intellectronica) [inline docs/index.rst]: especially

File: docs/introduction/basic_usage.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+###########
+Basic usage
+###########
+
+[describe the steps a user needs to do to get the system into a usable state for someone
```
Comment (intellectronica) [inline docs/introduction/basic_usage.rst]: That's a TODO I guess? Maybe add a comment to that effect - that way someone else can pick it up if they get to it before you.

**Comment by intellectronica** [comment]
:+1: Consider adding TODO comments next to relevant sections.

File: docs/introduction/basic_usage.rst (hunk 2 above)
Comment (evildmp) [inline docs/introduction/basic_usage.rst]: we have ``.. todo::`` directives in Sphinx!

Comment (evildmp) [inline docs/introduction/basic_usage.rst]: @intellectronica Oh I see, Paulo has (correctly) grabbed the example content from the application documentation starter pack, and that's what we're going to need to replace with the real thing.

**Comment by evildmp** [comment]
@czpython I have reworked the documentation configuration so we no longer need to build an entire environment just for the docs. Next step is to get the docs building and testing automagically on Travis; @mkoistinen is looking at that.

**Comment by czpython** [comment]
@evildmp but is this ok to merge in the meantime?

**Comment by evildmp** [comment]
@czpython yes go ahead and merge

**Pull request merged**